core/unix/os_exports.h

```diff
@@ -193,14 +193,15 @@ extern uint android_tls_base_offs;
 #endif
 
 #ifdef RISCV64
-/* FIXME i#3544: We might need to re-use ARM's approach and store DR TLS in
- * tcb_head_t::private field: typedef struct
+/* Re-using ARM's approach and store DR TLS in tcb_head_t::private,
+ * with the only difference being tp register points at the end of TCB
```

Comment:
nit: Add a period to end this sentence (and maybe also add a blank line after it -- right now it just runs right into the typedef which without any separation at all which is confusing)